modules/serverless-rn-1-16-0.adoc

```diff
@@ -0,0 +1,116 @@
+[id="serverless-rn-1-16-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.16.0
+
+[id="new-features-1-16-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.22.0.
+* {ServerlessProductName} now uses Knative Eventing 0.22.0.
+* {ServerlessProductName} now uses Kourier 0.22.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.22.0.
+* {ServerlessProductName} now uses Knative Kafka 0.22.0.
+* The `kn func` CLI plugin now uses `func` 0.16.0.
+* The `kn func emit` command has been added to the functions `kn` plugin. This command allows you to send events to test locally deployed functions.
+
+[id="fixed-issues-1-16-0_{context}"]
+== Fixed issues
+
+[id="known-issues-1-16-0_{context}"]
+== Known issues
+
+// Add a note about the following to the mTLS docs when they're ready to merge
+* If Service Mesh is enabled with mTLS, metrics for Knative Serving are disabled by default because Service Mesh prevents Prometheus from scraping metrics.
++
+If you want to enable Knative Serving metrics for use with Service Mesh and mTLS, you must complete the following steps:
```

Comment:
It would be a better, kinder user experience to include the commands to edit and apply these YAML files. If you're confident that your target user will be able to locate all of these (especially the Service Mesh control plane object), feel free to ignore this suggestion.

Reply:
I think it's OK for now since things like the Service Mesh control plane object are documented in more depth elsewhere